webapp/src/pages/UserPage/UserPage.css

```diff
@@ -2,6 +2,14 @@
   display: contents;
 }
 
+.user-page-loading {
```

Comment:
question: There is a loading class in the `AvatarPageTemplate`. Maybe we could remove this one?

Reply:
You are right, this is a leftover of some tests I did before.